### DISCUSSION TITLE
Suggest edits feature to propose changes to documentation

### DISCUSSION THREAD

**Discussion by manojradhakrishnan** [GitbookIO/community#446] [Feature Requests]
It would be great if Gitbook has a "suggest edits" feature to accept changes from users of the documentation

The users of the document, would not generally have access to the Gitbook editor, and hence getting their inputs is quite difficult without "suggest edits" feature. There is a similar discussion #395 , but that seems to be a feature for the maintainers of the documentation and not for the users

**Comment by aikoii** [comment]
Hi @manojradhakrishnan! Thanks for sharing your feedback.

We have a similar suggestion here #267, I encourage you to vote for it 🙂.

I must say, we don't have any immediate plans to support this yet.

(I will be closing this in favor of the first request)
